targets/csharp/source/source/PlayFabGameserverSDK.cs

```diff
@@ -0,0 +1,203 @@
+﻿// GSDK Only has the following supported frameworks for now
+#if NETSTANDARD2_0 || NETCOREAPP2_1
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PlayFab
+{
+    public enum GameState
```

Comment:
We might want to write minor comments here about the states.

Reply:
good call

---
In reply to: [255767990](https://github.com/PlayFab/SDKGenerator/pull/398#discussion_r255767990) [](ancestors = 255767990)